apps/admin-portal/src/store/actions/authenticate.ts

```diff
@@ -135,39 +135,40 @@ export const getProfileInformation = () => (dispatch): void => {
 export const handleSignIn = () => (dispatch) => {
     const oAuth = OAuth.getInstance();
     oAuth
-		.initialize({
-			baseUrls: [window["AppUtils"].getConfig().serverOrigin],
-			callbackURL: window["AppUtils"].getConfig().loginCallbackURL,
-			clientHost: window["AppUtils"].getConfig().clientOriginWithTenant,
-			clientID: window["AppUtils"].getConfig().clientID,
-			enablePKCE: true,
-			responseMode: process.env.NODE_ENV === "production" ? "form_post" : null,
-			scope: ["SYSTEM", "openid"],
-			serverOrigin: window["AppUtils"].getConfig().serverOriginWithTenant
-		})
-		.then(() => {
-			oAuth
-				.signIn()
-				.then((response) => {
-					dispatch(
-						setSignIn({
-							// eslint-disable-next-line @typescript-eslint/camelcase
-							display_name: response.displayName,
-							email: response.email,
-							scope: response.allowedScopes,
-							username: response.username
-						})
-					);
-
-					dispatch(getProfileInformation());
-				})
-				.catch((error) => {
-					throw error;
-				});
-		})
-		.catch((error) => {
-			throw error;
-		});
+        .initialize({
+            baseUrls: [window["AppUtils"].getConfig().serverOrigin],
+            callbackURL: window["AppUtils"].getConfig().loginCallbackURL,
+            clientHost: window["AppUtils"].getConfig().clientOriginWithTenant,
+            clientID: window["AppUtils"].getConfig().clientID,
+            enablePKCE: true,
+            responseMode: process.env.NODE_ENV === "production" ? "form_post" : null,
+            scope: ["SYSTEM", "openid"],
```

Comment:
Shall we get `"SYSTEM"` from `@wso2is/core/constants`, `TokenConstants` class? And also, aren't we enforcing `openid` scope?

```suggestion
            scope: [TokenConstants.SYSTEM_SCOPE, "openid"],
```